src/data/queries/qaGroup.ts

```diff
@@ -0,0 +1,21 @@
+import { QueryFormatter } from 'next-drupal-query'
+import { queries } from '.'
+import { ParagraphQaGroup } from '@/types/drupal/paragraph'
+import { QaGroup } from '@/types/formatted/qaGroup'
+
+export const formatter: QueryFormatter<ParagraphQaGroup, QaGroup> = (
+  entity: ParagraphQaGroup
+) => {
+  return {
+    questions:
+      entity.field_q_as.map?.((item) => ({
+        title: item.title,
+        answer: queries.formatData('paragraph--wysiwyg', item.field_answer),
```

Comment:
It looks like `field_answer` can be `paragraph--rich_text_char_limit_1000`:

```
"field_answer": {
  "data": {
    "type": "paragraph--rich_text_char_limit_1000",
    "id": "4a228710-bd01-4107-ac29-dc8411400039",
    "meta": {
      "target_revision_id": 769238,
      "drupal_internal__target_id": 13976
    }
  },
  "links": {
    "related": {
      "href": "https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/jsonapi/node/q_a/9299a086-e253-4ef4-a483-16057b7d43a0/field_answer?resourceVersion=id%3A697026"
    },
    "self": {
      "href": "https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/jsonapi/node/q_a/9299a086-e253-4ef4-a483-16057b7d43a0/relationships/field_answer?resourceVersion=id%3A697026"
    }
  }
},
```

This is effectively the same thing as `paragraph--wysiwyg`, but perhaps we can just use `formatParagraph` here and take advantage of that abstraction and not having to define the type of paragraph at this layer.

Reply:
I also wonder if we should just be formatting this entire thing using `queries.formatData('questionAnswer`, ...)`. But I'm not sure that query is properly set up yet.